static-router-config

```diff
@@ -20,11 +20,12 @@ then
   exit 1
 fi
 
-AWS_ADDRESSES_TO_USE=$(grep --color -A2 'Address' $1 | sed 's/.*:\ \(.*\)$/\1/' | grep '^[0-9].*$' | sed 's/^\(.*\)\/.*$/\1/')
-read -a AWS_ADDRESSES_TO_USE <<< $AWS_ADDRESSES_TO_USE
+AWS_ADDRESSES_TO_USEt=$(grep --color -A2 'Address' $1 | sed 's/.*:\ \(.*\)$/\1/' | grep '^[0-9].*$' | sed 's/^\(.*\)\/.*$/\1/')
+readarray -t AWS_ADDRESSES_TO_USE <<< $AWS_ADDRESSES_TO_USEt
```

Comment:
Typo fixed. 

Regarding _read_ -a vs _readarray_ [1], the only difference I see is that _read_ overwrites any previous data. I think in some bash versions it doesn't work as it used to. In my case your code doesn't run on GNU bash, version 4.4.11(1)-release (x86_64-pc-linux-gnu), that why I had to changed it and now it works like a charm.

[1] https://www.gnu.org/software/bash/manual/bashref.html#Arrays